bignumber-card/README.md

```diff
@@ -21,6 +21,8 @@ Severity object
 | red | number | **Required** | Value from which to start red color
 | green | number | **Required** | Value from which to start green color
 | amber | number | **Required** | Value from which to start amber color
+| normal | number | **Required** | Value from which is classed as a normal color N.B. this cannot be used with red/green/amber
```

Comment:
This should not be required. There should be a note that you should always have exactly three different severities.

Example: (amber, red, normal) or (green, normal, red) etc.

Reply:
would green, and normal not be the same thing? I assumed normal would give it a green band and then when it wondered off "normal" it changed to a red for example. So does normal display as a white background then?

Reply:
Blue (primary color of hass actually)